### DISCUSSION TITLE
Map field for locating object. | Directus 9.1.2

### DISCUSSION THREAD

**Discussion by BerkantCiftci** [directus/directus#10163] [Archive]
Hey guys! First of all really great this new version of Directus! I really appreciate your work! For my assignment I want to use the new map field. Now I don't know exactly how the point type works in map. I want to use this map to show where an object is located. So this object sends a lat and long. The map should then show where this object is located. Does point mean that I have to pass the lat and long in one float variable? And will the map then simply display this object? 

Thank you so much.

**Comment by rijkvanzanten** [comment]
Heya! That's exactly what a Point is 🙂 It's a single point on a map, using a lat-long to find the location.

The Directus API uses GeoJSON for its spatial data IO, so in this case the value for the map field would be

```json
{
    "type": "Point",
    "coordinates": [125.6, 10.1]
}
```

(from https://geojson.org)

**Comment by jyhro** [comment]
Hello, I want to know how can I separate lat and long into two different fields?
